include/hermes/BCGen/HBC/BytecodeProviderFromSrc.h

```diff
@@ -173,7 +177,8 @@ class BCProviderFromSrc final : public BCProviderBase {
       const ScopeChain &scopeChain,
       SourceErrorManager::DiagHandlerTy diagHandler = nullptr,
       void *diagContext = nullptr,
-      const std::function<void(Module &)> &runOptimizationPasses = {});
+      const std::function<void(Module &)> &runOptimizationPasses = {},
+      llvh::Optional<BytecodeGenerationOptions> bytecodeGenerationOptions = {});
```

Comment:
nit: we can make this more explicit
```suggestion
      llvh::Optional<BytecodeGenerationOptions> bytecodeGenerationOptions = llvh::None);
```

Reply:
cool, I I was wondering if there was an equivalent of `std::nullopt` for `llvh::Optional`